Symfony/CS/Linter/Linter.php

```diff
@@ -106,7 +106,8 @@ private function createProcessForFile($path)
             return $this->createProcessForSource(file_get_contents($path));
         }
 
-        $process = new Process(sprintf('%s -l %s', $this->executable, ProcessUtils::escapeArgument($path)));
+        $executable = ProcessUtils::escapeArgument($this->executable).(defined('HHVM_VESION') ? '--php' : '');
```

Comment:
I don't really trust symfony's implementation here given that the find method doesn't work correctly if the path needs escaping and you're running anything that needs params.